browser-extension/addTournesolRecommendations.js

```diff
@@ -139,20 +139,26 @@ const getTournesolComponent = () => {
       video_thumb.src = `https://img.youtube.com/vi/${video.video_id}/mqdefault.jpg`;
       thumb_div.append(video_thumb);
 
-      // TODO: This is commented out until our backend serves again the video duration in the metadata
-      // const video_duration = document.createElement('p');
-      // video_duration.setAttribute('class', 'time_span');
-
-      // Remove useless '00:' as hour value (we keep it if it is as minute value)
-      // var formatted_video_duration = video.duration;
+       const video_duration = document.createElement('p');
+       video_duration.setAttribute('class', 'time_span');
+
+      // Convert SECONDS to hh:mm:ss or mm:ss format depending on the duration
+      if (video.duration<3600) {
+      var formatted_video_duration = new Date(video.duration * 1000).toISOString().substring(14, 19)
+      } else {
+        var formatted_video_duration = new Date(video.duration * 1000).toISOString().substring(11, 19)
+      }
```

Comment:
You can greatly simplify the code with:

```suggestion
      const formatted_video_duration = new Date(video.duration * 1000).toISOString().substring(
          video.duration<3600 ? 14 : 11, 19
      )
```

Reply:
That probably scores well on the "relaxing" criteria :)
Nearly 3M views. How is it possible?